tasks/pyproject.toml

```diff
@@ -19,13 +19,14 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = "^3.10"
 robocorp-log = ">=2.4,<3"
 psutil = "^5.0"
 docstring_parser_fork = "^0.0.5"
 
 [tool.poetry.group.dev.dependencies]
 robocorp-devutils = {path = "../devutils/", develop = true}
+robocorp-truststore = "^0.8.0"
```

Comment:
Adding this here in order to avoid the warnings we'd see in the absence of such a dep during the run of the Invoke commands. (locally and maybe even in CI)

Didn't add it to the normal deps section as I don't want to have it interfering with the main **robocorp-truststore** we're bringing through Conda in the _conda.yaml_.

Reply:
I'm not sure we should still do this. See: https://sema4ai.slack.com/archives/C04SAQERGCF/p1709831458925959 for details.